omod/src/main/webapp/resources/scripts/fragments/visitDetails.js

```diff
@@ -63,7 +63,7 @@ function loadTemplates (visitId, patientId, fromEncounter, encounterCount, curre
                     });
                 }
                 
-            }).error(function(err) {
+            }).fail(function(err) {
```

Comment:
Is this change related to what this ticket is all about?

Reply:
The deprecation of success() to done() came together with the deprecation of error () to fail().....so it seemed logical to do both in the same sitting....What would be your suggestion?....Should I modified the ticket description to accommodate both or I shud have a different ticket for that too or I ignore it all together?

Reply:
@Ruhanga do you mean this?

Reply:
Ahh, yes. Nice catch. Thanks I hadn't see this change.